config/database.yml

```diff
@@ -1,25 +1,19 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: reloj_development
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
 test:
   <<: *default
-  database: db/test.sqlite3
+  database: reloj_test
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  database: reloj_production
```

Comment:
This file should not appear on GitHub, instead there should be an example file: `database.yml.example`

Reply:
@YilKanda ok thanks, i think that's fixed now 😀